### ISSUE TITLE
invalid value for Timezone `America/Ciudad_Juarez`

### ISSUE THREAD

**Issue by cobolbaby** [greenplum-db/gpdb#17144]
### Greenplum version or build

- 6.26.3

### Actual behavior

```sql
-- PG 12.x

postgres=# SELECT * FROM pg_timezone_names where name = 'America/Ciudad_Juarez';
         name          | abbrev | utc_offset | is_dst 
-----------------------+--------+------------+--------
 America/Ciudad_Juarez | MST    | -07:00:00  | f
(1 row)

-- GP
postgres=# SELECT * FROM pg_timezone_names where name = 'America/Ciudad_Juarez';
 name | abbrev | utc_offset | is_dst 
------+--------+------------+--------
(0 rows)
```

**Comment by kainwen** [comment]
This is not a bug. Postgres recently commit https://github.com/postgres/postgres/commit/2c95d8776c6a4d7b595e4446f74bdd7fcbdf8348
this commit for PG12.x, are since 12.14

```
gpadmin@gpdev:~/workspace/postgres/src/timezone/data$ git tag --contains 2c95d8776c6a4d7b595e4446f74bdd7fcbdf8348
REL_12_14
REL_12_15
```

Look at pg 12.14's release notes here: https://www.postgresql.org/docs/release/12.14/
you can find related update:

> Update time zone data files to tzdata release 2022g for DST law changes in Greenland and Mexico, plus historical corrections for northern Canada, Colombia, and Singapore.
> Notably, a new timezone America/Ciudad_Juarez has been split off from America/Ojinaga.

GP6's kernel version is PG9.4 and GP7's kernel version now is PG12.12, so this is expected behavior not a bug.

**Comment by cobolbaby** [comment]
@kainwen Is it possible to port the feature to version 6.x?

**Comment by ashwinstar** [comment]
@kainwen yes I think we need to incorporate/update to these new timezones files for 6X and 7X. Hence, should keep this open.
